media-fonts/adwaita-fonts/adwaita-fonts-48.2.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit font meson
+
+DESCRIPTION="Adwaita Sans:Variation of Inter, Adwaita Mono:Iosevka customized to match Inter"
```

Comment:
I just might not be sufficiently familiar with fonts to know what `Inter` or `Iosevka` mean -- maybe this is a good description. I don't know. I definitely have a hard time parsing it, especially with the `:` punctuation.

Maybe it'd just be better to go with the upstream repo's description, which is `The typefaces for GNOME`.